pkg/await/awaiters.go

```diff
@@ -567,7 +567,7 @@ func untilCoreV1ServiceAccountInitialized(c createAwaitConfig) error {
 	// secrets array (i.e., in addition to the secrets specified by the user).
 	//
 
-	specSecrets, _ := openapi.Pluck(c.currentInputs.Object, "secrets")
+	specSecrets, _ := openapi.Pluck(c.currentOutputs.Object, "secrets")
```

Comment:
Why the change from inputs to outputs?

Reply:
I had the same question - but just assumed that it was me not having enough context as a novice